### ISSUE TITLE
After 2.1 update, Instagram widget stopped working

### ISSUE THREAD

**Issue by cialu** [tagDiv/ionMag#15]
Hi.
I have updated the theme to the latest version and after that, the Instagram widget stopped working. If I leave my Instagram account, the widget shows old pictures and not refresh. If I try with a different account username, the widget shows: "Instagram data error: Instagram data is not set, please check the ID".

What can I do to make it working again?

Thanks in advance.

**Comment by tripvisits** [comment]
cialu  is it solved? I have this problem

**Comment by cialu** [comment]
@tripvisits I'm sorry but the issue is still remaining. I gave a look at the code and I did many trials, but can't find a solution.  No way to solve this, so I'm waiting for a suggestion from the theme developers.

**Comment by tripvisits** [comment]
Thank you so much Luca. I m using instagram feed plugin. I hope it will be
fix...

Thank u

Best Regards

2018-04-10 18:05 GMT+03:00 Luca Ciavatta <notifications@github.com>:

**Comment by ralphskie** [comment]
I'm also experiencing this one. Hopefully the devs can take a look at this issue.

**Comment by cialu** [comment]
@ralphskie I was working on the same file but without success, too many callbacks and too much code to look into. Anyway, I'm trying your gist, no errors but the widget shows nothing. Just a Follow us on Instagram 'instagram_id' and nothing more. I'll check what you did and I'll comment on the gist if I found something.

**Comment by cialu** [comment]
@ralphskie I had to remove the cache and now it's working again. Thank you so much. I'll wait to close the issue until the solution goes upstream.

**Comment by ralphskie** [comment]
@cialu yes manually deleting the cache on the database will trigger an http request on instagram, thus getting a new data format then save to the cache and display on the page.
Good to know it works fine on you. :smile:

**Comment by kaiofreitas** [comment]
How do you delete the cache on the database?

**Comment by ralphskie** [comment]
im manually deleting it.
1. go to your database -> "options" tables, if you have prefix then "prefix_options"
2. see "td_012_remote_cache" on the "option_name" field
3. delete/clear/blank the "option_value" of that row.

now refresh the page where your instagram widget is.

**Comment by kaiofreitas** [comment]
Thanks! It worked!
